internal/permission/implementation.go

```diff
@@ -1,12 +1,12 @@
 package permission
 
 type DataProvider interface {
-	get(fqfields []Fqfield) map[Fqfield]Value
+	Get(fqfields []Fqfield) map[Fqfield]Value
```

Comment:
Only Methods with a capital letter are exported. An interface with an unexported method can not be implemented by another package.